4-projects/README.md

```diff
@@ -7,8 +7,10 @@ The purpose of this step is to setup folder structure and projects for applicati
 1. 0-bootstrap executed successfully.
 1. 1-org executed successfully.
 1. 2-environments executed successfully.
-1. 3-networks executed successfully.
+1. 4-projects executed successfully.
```

Comment:
`3-networks` must be executed successfully.

this step is the deploy of `4-projects `